### PR TITLE
Real data in home item table

### PR DESCRIPTION
**Pull request by hobbitronics**
## Fixed
- Replaced the non-exisitng properties in the home item table

Screen Shot 2021-09-09 at 3 39 18 PM

### CODE REVIEW

File: src/pages/home.svelte [hunk 1]
```diff
@@ -35,6 +35,13 @@ onMount( () => {
   loadPolicies()
 })
 
+const getItemName = item => item.name || ''
```
Comment (Schparky) [inline src/pages/home.svelte:38]: If you're going to make these little getter functions, should they live somewhere where they can be exported (e.g. data/items.js)?

**Review by Schparky — APPROVED** [review]

File: src/pages/home.svelte (hunk 1 above)
Comment (hobbitronics) [reply src/pages/home.svelte:38]: The issue Ihad is the item in each row is in an #each loop from items, so I couldn't do a reactive statement. We would need a findItemById function as well. Do you think its worth it or should I just put it back in the markup?

Comment (Schparky) [reply src/pages/home.svelte:38]: I guess I was thinking to just move these as-is but no big deal either way.

**Comment by hobbitronics** [comment]
Oh, yeah that would work. I had my brain off for a second.

File: src/pages/home.svelte (hunk 1 above)
Comment (hobbitronics) [reply src/pages/home.svelte:38]: oh yeah, that works. I just had my brain switched off.

**Pull request merged**